### ISSUE TITLE
Alt+Space opens system menu instead of generating input

### ISSUE THREAD

**Issue by mbunkus** [alacritty/alacritty#7865]
On Windows I cannot get Alt+Space to send a key code. Instead the system menu is always opened. This is important insofar as Emacs uses Alt+Space. Other terminal emulators (putty, kitty, Windows Terminal) work fine with Alt+Space.

I've tried re-binding via the following, but to no avail:

```
[keyboard]

bindings = [
  { key = "Space", mods = "Alt", action = "ReceiveChar" }
]
```

### System

OS: Windows 11
Version: `alacritty 0.13.2 (bb8ea18)`

### Logs

Available [here](https://gist.github.com/mbunkus/e5e090292555f564ab38d31fe1c36956). I started Alacritty, pressed Alt, held Alt, pressed Space, the window's system menu appears. I release both keys, close Alacritty.

**Comment by chrisduerr** [comment]
Is this not a standard binding on Windows?

**Comment by mbunkus** [comment]
It is. However, programs can & do intercept this combination in order to do anything. For example, the Windows Terminal, Putty & kitty all intercept it & pass it through to the running application (in my case: ssh & Emacs inside ssh). The key sequence generated is ESC space (`^[ ` or 0x1b 20).

**Comment by mbunkus** [comment]
And before you ask, a binding inserting those chars doesn't work either. I've tried that already:

```
[keyboard]
bindings = [
  # other bindings, too
  { key = "Space", mods = "Alt", chars = "\u001b\u0020" }
```

The window's system menu opens.

**Comment by chrisduerr** [comment]
> And before you ask, a binding inserting those chars doesn't work either.

Yes, system bindings are handled before keybindings. So this is to be expected.

@kchibisov Winit has no way to configure system bindings at this point, correct?

**Comment by kchibisov** [comment]
>@kchibisov Winit has no way to configure system bindings at this point, correct?

I think it's more like a _hittest_ situation, we'll support it in the future, but not now...

**Comment by chrisduerr** [comment]
I'll just mark it as a new feature for now, though we probably have some similar ones in our backlog.
